scripts/wsrep_sst_clone.sh

```diff
@@ -996,7 +1000,7 @@ EOF
 
     wsrep_log_info "Launching clone recipient daemon"
     wsrep_log_info "-> using: $CLONE_ENV $CLONE_BINARY_SAFE $DEFAULT_OPTIONS "
-    wsrep_log_debug "-> Test connection as: -u$CLONE_USER -p$CLONE_PSWD -h $JOINER_CLONE_HOST -P $JOINER_CLONE_PORT"
+    wsrep_log_debug "-> Test connection as: -u$CLONE_USER -pCLONE_PSWD -h $JOINER_CLONE_HOST -P $JOINER_CLONE_PORT"
```

Comment:
instead of `-pCLONE_PSWD`, please use -p*******

Reply:
done